internal/marketplaces/subscriptions/service.go

```diff
@@ -172,15 +172,25 @@ func ensureBundleExists(
 	namespace, bundleName string,
 ) (uuid.UUID, error) {
 	// This is a no-op if this namespace/name pair already exists
-	dbBundle, err := qtx.UpsertBundle(ctx, db.UpsertBundleParams{
+	err := qtx.UpsertBundle(ctx, db.UpsertBundleParams{
 		Namespace: namespace,
 		Name:      bundleName,
 	})
 	if err != nil {
 		return uuid.Nil, err
 	}
 
-	return dbBundle.ID, nil
```

Comment:
ah, good ol' `on conflict do nothing` bug? I think this is fine, but IIRC in other queries we've just rewrote a random attribute to the same name to get a consistent return.

Reply:
Yep, that's the issue exactly. I'll have a think about making it nicer in future.